src/arbiter.rs

```diff
@@ -196,7 +196,7 @@ impl Arbiter {
         });
     }
 
-    /// Executes a future on the current thread.
+    /// Executes a lazily constructed future on the current thread.
```

Comment:
I think it would be more accurate to describe it as:

Executes a function and a future returned by it on the current thread

Reply:
I deem "the lazily constructed" wording as accurate, albeit maybe a bit short or even cryptic to some. I've instead expanded on that a bit.